doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,149 @@
+.. _ug_zigbee_configuring_zboss_traces:
+
+Configuring ZBOSS traces in |NCS|
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This ZBOSS stack comes in a set of precompiled libraries which can complicate the debugging process.
+To help with that, the ZBOSS stack can be configured to print trace logs that allow you to trace the stack behavior.
+This page describes how to enable and configure ZBOSS trace logs.
+
+Trace logs are printed in binary form and require access to stack source files, which are to be decoded into log messages.
+Enabling trace logs can help with the debugging process even if you cannot decode them (due to lack of access to stack source files) because of the information they can provide about the stack behavior whenever an issue is found or reproduced.
+
+
+.. note::
+     ZBOSS trace logs are meant to be used for debugging ZBOSS based applications.
+     These libraries are not certified and should not be used for production firmware or end products.
+
+
+
+Requirements
+************
+
+To collect and view the log files, you will need software that can do the following:
+
+* Allow for reading serial ports
+  For example, if you are using Windows you can use PuTTY. (See :ref:`putty` for more information.)
+
+* Find the correct COM port
+  One way do this is by setting the Kconfig option :kconfig:`CONFIG_USB_DEVICE_PRODUCT="Zigbee sample"` to help identify the device in the system's :guilabel:`Device Manager`.
+
+
+.. rst-class:: numbered-step
+
+Switch to ZBOSS libraries with compiled-in trace logs
+*****************************************************
+
+Set the Kconfig option :kconfig:`CONFIG_ZIGBEE_ENABLE_TRACES` to switch to ZBOSS libraries with compiled-in trace logs.
+
+The ZBOSS stack comes in a precompiled form and trace logs are not compiled-in by default.
+An additional set of ZBOSS libraries are available in nrfxlib, which does have trace logs compiled-in.
+
+
+.. rst-class:: numbered-step
+
+Select which ZBOSS trace logs to print
+**************************************
+
+1. Select which subsystems to receive logs from by configuring the ZBOSS trace mask with Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_MASK`.
+   Trace masks can be created by adding masks of subsystems to receive trace logs from.
+   For available subsystems, see :file:`zb_trace.h`. (NOTE: CHANGE THIS TO PATH TO FILE, instead of just file name!)
```

Comment:
Need to find this file

Reply:
It's either `nrfxlib/zboss/production/include/zb_trace.h` or `nrfxlib/zboss/development/include/zb_trace.h`.
I think that we could stick with the `nrfxlib/zboss/production/include/zb_trace.h` but in both cases this is the exact same file.